README.md

```diff
@@ -253,49 +258,98 @@ compensation.
+    company’s board of directors decides to increase the number of outstanding shares by
+    doubling or tripling the number of shares each stockholder has.
+    Stock splits are a way to increase the liquidity of a company’s stock when the stock price
+    is too high for average buyers.
+    It does not increase the value (or "market capitalization") of the stock, because the
+    price per share is reduced by the same multiple by which the outstanding shares increase.
+    However, increased market presence can bring in new investors, ultimately raising the
+    price per share.
+  - [**Share buyback**](https://www.investopedia.com/terms/s/sharerepurchase.asp) (or share
+    repurchase) in contrast decreases the number of outstanding shares.
+    If a company thinks its stock is undervalued, it will buy back a certain number of shares
+    from the marketplace, thus increasing the value of each stockholder’s shares.
+  - There is a lot of nuance and complexity in all this, because each part is defined by piles
+    and piles of legal documents.
 - If you have stock, what ultimately matters is not the number of shares you hold, but the
   **percentage ownership** of the entire company that that number represents.
   To determine the percentage of the company a certain number of shares represents, you
   divide it by the number of outstanding shares.
   Even if you have a fixed number of shares, your percentage ownership will change over time
   as the outstanding shares change.
+- Stock is also a taxable source of income—sometimes stock is taxed even before you’ve made
+  money! Stocks and taxes are highly complex, till-death-do-they-part subjects.
+  We’ll get to the how and why of what you owe in our detailed sections on
+  [Taxes](#taxes-on-equity-compensation).
 
 ### What is equity?
 
-🚧 Structure: Intro on how equity is broader than just stock.
+As we discussed in the introduction to this Guide, equity is about more than just holding stock. It's a calculation of the value of ownership in a company, and its purpose is to align the interests of stockholders (in the case of equity compensation, the employees) with the financial and philosophical goals of a company. In many ways, equity is about risk: rising waters will lift all boats, but you can also go down with the ship.
 
 🄳 Informally, we say you have **equity** in a company when you have some kind of ownership
 or likely future ownership.
-In practice, this can mean, restricted stock, stock options, and RSUs, which we’ll be
-discussing at length.
+In order to encourage employee ownership, companies might offer employees restricted stock, different types of stock options, and RSUs (restricted stock units) as part of their compensation package. To a certain degree, employees who have equity in the form of these stocks or units have power over how, when, and whether they want to become part owners in their company. We'll be
```

Comment:
Not sure about this change, in particular the 2nd sentence, due to a few nits:
- Options are neither stocks nor units, but options.
- Having equity doesn't means you have a sort of possible or approximate ownership, but not that you have power over it.
- "To a certain degree" seems avoidably vague.